src/app/models/form.server.model.ts

```diff
@@ -529,6 +530,22 @@ const compileFormModel = (db: Mongoose): IFormModel => {
     return this.save()
   }
 
+  FormSchema.methods.updateMsgSrvcName = async function (
+    msgSrvcName: string,
+    session?: ClientSession,
+  ) {
+    this.msgSrvcName = msgSrvcName
+
+    session ? await this.save({ session }) : await this.save()
```

Comment:
do we need the conditional check? if it's undefined it'll still work as is. also suggest returning the .save function call so the updated document is returned (even if we do not use it). Might need a flag in .save to return the updated document.

```ts
return this.save({session}) // should work even if session is undefined
```

Reply:
Checked and tried, dosent need to :D